### DISCUSSION TITLE
Log IMU data

### DISCUSSION THREAD

**Discussion by sesem738** [bitcraze/discussions#974] [Q&A]
What's the best way to log IMU data?

**Comment by knmcguire** [comment]
The best way depends on what you want exactly :)

If you'd just like to look at realtime logs with plotting, then the [cfclient ](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/) with the [plotter tab](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/plotter_tab/) is the best.

There is also the [logblocks tab](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/logblocks_tab/) where you can choose to write logs to a file. 

If you'd like to make your own script out of it you can take a look at [the crazyflie python library examples](https://github.com/bitcraze/crazyflie-lib-python/tree/master/examples/logging). You'd still need to write something extra in there that will write the values in a script but there are plenty of examples for that online.

Also you can find a list the logging values you want to log here: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/logs/
